server/api/hook.go

```diff
@@ -205,20 +213,41 @@ func PostHook(c *gin.Context) {
 		return
 	}
 
-	if repo.UserID == 0 {
-		msg := fmt.Sprintf("ignoring hook. repo %s has no owner.", repo.FullName)
-		log.Warn().Msg(msg)
-		c.String(http.StatusNoContent, msg)
+	//
+	// 4. Now we are sure it's valid so we start to create / update repo information
```

Comment:
```suggestion
	// Update repo
```

It has no reference (key) so I would just leave it out